modules/storages/app/common/storages/peripherals/nextcloud_registry.rb

```diff
@@ -88,6 +88,7 @@ module Peripherals
       namespace("authentication") do
         register(:userless, StorageInteraction::AuthenticationStrategies::NextcloudStrategies::UserLess, call: false)
         register(:user_bound, StorageInteraction::AuthenticationStrategies::NextcloudStrategies::UserBound)
+        register(:bearer_token, StorageInteraction::AuthenticationStrategies::NextcloudStrategies::BearerToken)
```

Comment:
:green_circle: Naming-wise I am not the biggest fan of `bearer_token` / `BearerToken` here, because the `UserBound` strategy can also be correctly described with the term "Bearer Token", because it uses the `Authorization` HTTP header with a token of type `Bearer`. It's even thinkable that a future implementation of `UserLess` would do the same.

I.e. the name doesn't quite indicate what makes the new strategy special. Proposals:

* `Passthrough`
* `SpecificBearerToken`

Reply:
I'd go with `SpecificBearerToken` then. Thanks for suggesting!